analytics-service.js

```diff
@@ -0,0 +1,30 @@
+const dbClient = require('./database/db').client;
+const config = require('./config');
+const dbName = config.DATABASE_NAME;
+const collection = config.COLLECTION;
+
+class AnalyticsService {
+
+    
+    static async getAnalytics(req, res) {
+        try {
+            let result = await dbClient.db(dbName).collection(collection).find({type: req.params.type}).toArray()
```

Comment:
I would suggest you use`mongoose` as a mongo db client package. It is simpler and easier